Example/RSFormViewExampleApp/SignupExampleViewModel.swift

```diff
@@ -42,9 +49,19 @@ class SignupExampleViewModel: FormViewModel {
     
     return data
   }
+
+  func siblingsItem() -> FormItem {
+    return StepperCellItem(with: FormField(name: FieldName.numberOfSiblings.rawValue, //Identifier for the field
+                          initialValue: "10", //Provide a default value for the field
+                          placeholder: "",
+                          fieldType: .double(maxDecimalPlaces: 0), //field type describes the behaviour the field will expect and provide a default validation
```

Comment:
Shouldn't be `Int`?

Reply:
yes, it makes more sense in this example